packages/core/test/toast/toasterTests.ts

```diff
@@ -121,4 +121,17 @@ describe("Toaster", () => {
             "mutation side effect!",
         );
     });
+
+    describe("with autoFocus set to true", () => {
```

Comment:
no need to add a whole describe block here, just make it one more test in the block above.

also redundancy between `describe` and `it` here.